charts/openebs/README.md

```diff
@@ -147,3 +147,97 @@ helm install --name openebs -f values.yaml openebs/openebs
 ```
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Below charts are dependent charts of this chart
+-  [openebs-ndm](https://openebs.github.io/node-disk-manager)
+-  [localpv-provisioner](https://openebs.github.io/dynamic-localpv-provisioner)
+-  [cstor](https://openebs.github.io/cstor-operators)
+-  [jiva](https://openebs.github.io/jiva-operator)
+-  [zfs-localpv](https://openebs.github.io/zfs-localpv)
+-  [lvm-localpv](https://openebs.github.io/lvm-localpv)
+
+## Dependency tree of this chart
+```bash
+openebs
+├── openebs-ndm
+├── localpv-provisioner
+│   └── openebs-ndm
```

Comment:
n_: should we keep this as (optional). Similarly under Jiva. 

It is only required by cStor.

Reply:
done